pkg/domain/infra/abi/containers.go

```diff
@@ -380,17 +380,27 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 	}
 	names = tmpNames
 
-	ctrs, err := getContainersByContext(options.All, options.Latest, names, ic.Libpod)
+	ctrs, rawInputs, err := getContainersAndInputByContext(options.All, options.Latest, names, options.Filters, ic.Libpod)
 	if err != nil && !(options.Ignore && errors.Is(err, define.ErrNoSuchCtr)) {
 		// Failed to get containers. If force is specified, get the containers ID
 		// and evict them
 		if !options.Force {
 			return nil, err
 		}
 
+		ctrMap := map[string]string{}
```

Comment:
The name `ctrMap` is very generic. Can we give it a more expressive name, for instance, `idToRawInput`?